packages/pn-personafisica-webapp/src/pages/__test__/NotificationDetail.page.test.tsx

```diff
@@ -660,6 +660,38 @@ describe('NotificationDetail Page', async () => {
     vi.useRealTimers();
   });
 
+  it('getReceivedNotificationPaymentUrl is called in error and the number of payments does not change', async () => {
```

Comment:
Give the test a more meaningful description like: "should not duplicate payments when api call to cart respond with an error". Moreover check that the layout changes as we expect, showing the error message and the "reload" button!